hw/ip/alert_handler/doc/index.md

```diff
@@ -224,17 +219,17 @@ The wave pattern below illustrates differential full handshake mechanism.
   signal: [
     { name: 'clk_i',        wave: 'p...............' },
     { name: 'alert_i',      wave: '01.|..|..|...|..' },
-    { name: 'alert_p',      wave: '01.|..|0.|..1|..' , node: '.a.....c....e'},
-    { name: 'alert_n',      wave: '10.|..|1.|..0|..' },
-    { name: 'ack_p',        wave: '0..|1.|..|0..|1.' , node: '....b.....d..'},
-    { name: 'ack_n',        wave: '1..|0.|..|1..|0.' },
+    { name: 'alert_po/pi',  wave: '01.|..|0.|..1|..' , node: '.a.....c....e'},
```

Comment:
just a note that this is not rendering in the documentation today, but works on bubble. I'm going to assume this is simply the fact that @gkelly hasn't turned on the live updates yet, but keep an eye on it.

Reply:
yes that might be the case. will check again later.

Reply:
That's correct, sorry.  I'll work on getting that enabled.